src/icons/fileIcons.ts

```diff
@@ -2094,5 +2094,9 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'tree', fileExtensions: ['tree'] },
+    {
+      name: 'cadence',
+      fileNames: ['.cdc'],
```

Comment:
Probably you want to set the `fileExtensions` instead of `fileNames`:

```diff
-fileNames: ['.cdc'],
+fileExtensions: ['cdc'],
```

The difference is that fileNames only affects files which are exactly called as the string in the array. So you need to have a file called ".cdc". I guess that you want to apply the icon to files that just end with ".cdc" like "sample.cdc" right? If yes, please change that behavior to let it work properly :)

Reply:
Thanks! This is what I wanted, I changed it now.

Reply:
👍